geth/notification/fcm/notifier.go

```diff
@@ -0,0 +1,55 @@
+package fcm
+
+import (
+	"github.com/NaySoftware/go-fcm"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/notification"
+)
+
+// Notifier represents messaging provider for notifications.
+type Notifier struct {
```

Comment:
As I wrote before, it's not `Notifier` any more, it's `Notification`, right?

Reply:
Ok, i just think that we want only rename constructor.

Reply:
We want names to be compliant with entities essence :)